packages/next/src/views/Version/index.tsx

```diff
@@ -142,26 +143,29 @@ export async function VersionView(props: DocumentViewServerProps) {
     }
   }
 
-  const selectedLocales: OptionObject[] = []
+  let selectedLocales: OptionObject[] = []
   if (localization) {
+    let locales: Locale[] = localization.locales
```

Comment:
Ah, slight change, will need to assign `locales = localization.locales` in an else block if there are no `localeCodesFromParams`. Initialize locales to empty array.

Reply:
Like this?
<img width="602" alt="Screenshot 2025-02-28 at 4 18 15 PM" src="https://github.com/user-attachments/assets/0dd0828c-5d4e-4a1f-8fa9-efe41aa158e0" />

Reply:
@JessChowdhury yes like that 👍

Reply:
@JarrodMFlesch neat - changes pushed!